# Question
Look at this function:

```js
const bailOut = () => {
  for (let i = 0; i < 5; i++) {
    if (i > 3) return;
    console.log('Inside the loop', i);
  }
  console.log('Outside the loop');
};
```

Is that `console.log('Outside the loop');` ever going to run? Why or why not?

# Answer
It won't run because it's not in the same block of code as the `for` loop.

# Teacher Feedback

What happens when the keyword `return` is used? Because the `return` keyword is used, the function is exited so anything after this condition is met will not be read!

Score: 1/3